packages/edit-site/src/components/page-actions/trash-page-menu-item.js

```diff
@@ -34,39 +29,31 @@ export default function DeletePageMenuItem( { postId, onRemove } ) {
 			createSuccessNotice(
 				sprintf(
 					/* translators: The page's title. */
-					__( '"%s" deleted.' ),
+					__( '"%s" moved to the Trash.' ),
 					decodeEntities( page.title.rendered )
 				),
 				{
 					type: 'snackbar',
-					id: 'edit-site-page-removed',
+					id: 'edit-site-page-trashed',
 				}
 			);
 			onRemove?.();
 		} catch ( error ) {
 			const errorMessage =
 				error.message && error.code !== 'unknown_error'
 					? error.message
-					: __( 'An error occurred while deleting the page.' );
+					: __(
+							'An error occurred while moving the page to the trash.'
```

Comment:
```suggestion
							'An error occurred while moving the page to the Trash.'
```

WordPress core seems to capitalize when there is an article (the), what do you think?

![restored](https://github.com/WordPress/gutenberg/assets/54422211/521d144f-9924-46d6-81e5-869dabd77aa4)
![moved](https://github.com/WordPress/gutenberg/assets/54422211/32f7ee1c-aa89-4741-a2ae-9af27d9e4999)